Readme (#758)
Textualize/textual: README.md

# Textual

![Textual splash image](./imgs/textual.png)

> Textual is a Python framework for creating interactive applications within your terminal.

Reviewer: Currently sounds like you make the apps _using_ your terminal.
```suggestion
Textual is a Python framework for creating interactive applications that run in your terminal.
```

<details>
  <summary> 🎬 Code browser </summary>
  <hr>

  This is the [code_browser.py](./examples/code_browser.py) example which clocks in at 61 lines (*including* docstrings and blank lines).

  https://user-images.githubusercontent.com/554369/189394703-364b5caa-97e0-45db-907d-7b1620d6411f.mov
  
 </details>

## About

> Textual adds interactivity to [Rich](https://github.com/Textualize/rich) with a Python API inspired by modern development development.

Reviewer: ```suggestion
Textual adds interactivity to [Rich](https://github.com/Textualize/rich) with a Python API inspired by modern web development.
```

> On modern terminal software (installed by default on most system) Textual apps can use **16.7 million** colors with mouse support and smooth flicker-free animation. A powerful layout engine and re-usable components makes it possible to build apps that rival the desktop and web experience. 

Reviewer: ```suggestion
On modern terminal software (installed by default on most systems), Textual apps can use **16.7 million** colors with mouse support and smooth flicker-free animation. A powerful layout engine and re-usable components makes it possible to build apps that rival the desktop and web experience. 
```

## Compatibility

> Textual runs on Linux, MacOS, and Windows. Rich requires Python 3.7 or above.

Reviewer: Assuming this was a copy paste error
```suggestion
Textual runs on Linux, MacOS, and Windows. It requires Python 3.7 or above.
```
